theme/material/vaadin-message-styles.js

```diff
@@ -6,35 +6,34 @@ import '@vaadin/vaadin-avatar/theme/lumo/vaadin-avatar.js';
 registerStyles(
   'vaadin-message',
   css`
-    :host {
-      padding: 8px;
-    }
-
     :host {
       font-family: var(--material-font-family);
-      font-size: var(--material-small-font-size);
-    }
-
-    .vaadin-message-wrapper {
-      padding-left: 8px;
-      padding-right: 0;
+      font-size: var(--material-body-font-size);
+      line-height: 1.5;
+      padding: 0.875em 1em;
+      -moz-osx-font-smoothing: grayscale;
+      -webkit-font-smoothing: antialiased;
+      -webkit-text-size-adjust: 100%;
     }
 
-    :host([dir='rtl']) .vaadin-message-wrapper {
-      padding-left: 0;
-      padding-right: 8px;
-    }
-
-    .vaadin-message-header {
-      padding-bottom: 8px;
+    vaadin-avatar {
+      margin-inline-end: 0.5em;
+      margin-top: 0.125em;
     }
 
     [part='name'] {
-      font-weight: 500;
+      margin-inline-end: auto;
```

Comment:
This pushes the time element all the way to the right. In lumo, you used `var(--lumo-space-s)` instead of `auto`, and that resolves to `0.5rem`. Maybe it should also be `0.5em` as with avatar?

And as noted in another comment, use margin-left and margin-right instead of margin-inline-end.

Reply:
Yes, I think that's more in line with how MD wants it. Inspired by their Lists guideline, for example:
https://lh3.googleusercontent.com/A2wjQsvLCpJM3XEvVKxM9fZOaa2jY0BsPAg20bIjCNd7ETTzfxOzVXi0qskdhNaTFrZLKvwfNMYOefqGp3OTwJnFYtAgGLNVju3gd_o=w1064-v0

Reply:
Ah okay, so your comment about it being too far away was directed towards Lumo. Then all is fine.